staging/src/k8s.io/client-go/tools/cache/thread_safe_store.go

```diff
@@ -50,6 +51,7 @@ type ThreadSafeStore interface {
 	IndexKeys(indexName, indexKey string) ([]string, error)
 	ListIndexFuncValues(name string) []string
 	ByIndex(indexName, indexKey string) ([]interface{}, error)
+	ByIndexes(conds IndexConditions) ([]interface{}, error)
```

Comment:
This is also changing an existing interface. Wonder if we should have a new interface as well.